core/src/impl/Kokkos_Utilities.hpp

```diff
@@ -120,6 +120,10 @@ struct is_specialization_of : std::false_type {};
 template <template <class...> class Template, class... Args>
 struct is_specialization_of<Template<Args...>, Template> : std::true_type {};
 
+template <typename T, template <typename...> class U>
+inline constexpr bool is_specialization_of_v =
+    is_specialization_of<T, U>::value;
+
```

Comment:
I think this should be a separate pr

Reply:
Given that `is_specialization_of` is only used in the graph stuff, I would say that it's OK not to make a separate PR with just this change. Up to you.

Reply:
yeah, it is probably fine

Reply:
Thanks, closing then.